tests/atlas.phpt

```diff
@@ -7,6 +7,56 @@ Atlas Connectivity Tests
 <?php
 require_once __DIR__ . "/utils/basic.inc";
 
+function extractUri(string $env): ?string
+{
+	return getenv($env) ?: null;
+}
+
+function extractUriWithCertificate(string $env): ?array
+{
+	$uri = getenv($env);
+	if (! is_string($uri)) {
+		return null;
+	}
+
+	$cert = getenv($env . '_CERT_BASE64');
+	if (! is_string($cert)) {
+		return null;
+	}
+
+	$certPath = tempnam(sys_get_temp_dir(), 'cert_');
+	$certContents = base64_decode($cert);
+	if (! $certPath || ! $certContents) {
+		return null;
+	}
+
+	file_put_contents($certPath, $certContents);
+	chmod($certPath, 0600);
```

Comment:
Is this necessary? [`tempnam()`](https://www.php.net/tempnam) should already create the file with these perms.